doc/developer-notes.md

```diff
@@ -91,6 +91,9 @@ code.
   - Compile-time constant names are all uppercase, and use `_` to separate words.
   - Class names, function names, and method names are UpperCamelCase
     (PascalCase). Do not prefix class names with `C`.
+  - Enumerators may be snake_case, PascalCase, or ALL_CAPS. This is a more tolerant policy than
```

Comment:
In commit "doc: update enum naming in developer notes" (2d813c42ed5573d3afd80d677e2ded41e8d5b7e8)

I'd probably insert the new bullet point one line up so the two constant guidelines are next to each other:

- Variable and namespace names
- Constant names
- Enumerator constant names
- Class names, method names, and function names
- Test names

(This might require rebasing after #18568 to avoid conflicts)

Reply:
Thanks for the heads up on rebasing. Done.